DashAI/back/dataloaders/classes/dataset_dashai.py

```diff
@@ -0,0 +1,151 @@
+import json
+import os
+from typing import Dict, List
+
+from datasets import ClassLabel, Dataset, Value, load_from_disk
+
+
+class DatasetDashAI(Dataset):
+    def __init__(
+        self,
+        table,
+        inputs_columns: List[str],
+        outputs_columns: List[str],
+        *args,
+        **kwargs,
+    ):
+        """
+        Dataset from Hugging Face with more metadata about dataset.
+        Args:
+            table: Table
+            inputs_columns: list
+            outputs_columns: list
+            *args: arguments forwarded to super.
+            **kwargs: keyword arguments forwarded to super.
```

Comment:
The docstring does not follow the numpy convention:

```suggestion
        Parameters
        ----------
        table: 
            <some description>
        inputs_columns: list
            <some description>
        outputs_columns: list
            <some description>
```

It is not necessary to document `*args` or `*kwargs`.